src/main/java/org/jabref/gui/preferences/general/GeneralTabViewModel.java

```diff
@@ -54,6 +55,9 @@ public class GeneralTabViewModel implements PreferenceTabViewModel {
     private final ReadOnlyListProperty<ThemeTypes> themesListProperty =
             new ReadOnlyListWrapper<>(FXCollections.observableArrayList(ThemeTypes.values()));
     private final ObjectProperty<ThemeTypes> selectedThemeProperty = new SimpleObjectProperty<>();
+
+    private final BooleanProperty flagWhenAutomaticButtonSelected = new SimpleBooleanProperty();
```

Comment:
Why such a long name? The other names are shorter?

Proposal: 

    automaticSyncActivatedProperty